pkg/resolver/search.go

```diff
@@ -144,17 +157,38 @@ func (s *SearchResult) resolveCount(query string, args []interface{}) (int, erro
 	return count, err
 }
 
+func (s *SearchResult) resolveUids(query string, args []interface{}) ([]*string, error) {
+	rows, err := s.pool.Query(context.Background(), query, args...)
+	if err != nil {
+		klog.Errorf("Error resolving query [%s] with args [%+v]. Error: [%+v]", query, args, err)
+	}
+	var uid string
+
+	defer rows.Close()
+	for rows.Next() {
+		err = rows.Scan(&uid)
+		if err != nil {
+			klog.Errorf("Error %s retrieving rows for query:%s", err.Error(), query)
+		}
+		s.uids = append(s.uids, &uid)
+	}
+
+	return s.uids, err
```

Comment:
There's no need to return `s.uids` from this function because you have saved it in the SearchResult object, so other functions can access it from there.